src/solver/opamCudf.ml

```diff
@@ -968,7 +1045,13 @@ let extract_explanations packages cudfnv2opam reasons : explanation list =
         | `Missing (_, sdeps', fdeps') -> sdeps = sdeps' && fdeps = fdeps'
         | _ -> false)
   in
+  log ~level:3 "Explanations: %a" Pp_explanation.pp_explanationlist explanations;
   match explanations with
+  | [] ->
+    OpamConsole.error_and_exit `Internal_error
+      "Internal error while computing conflict explanations:\n\
+       sorry about that. Please report how you got here in \
+       https://github.com/ocaml/opam/issues/4373 if possible."
```

Comment:
I don't agree having such url in an error message. Just redirecting them to open an issue is sufficient, or point to the issue tracker if you want to set an url.

Reply:
how is it now?